test/Mono.Linker.Tests.Cases.Expectations/Metadata/RequiresDynamicCodeAttribute.cs

```diff
@@ -1,41 +0,0 @@
-﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-#nullable enable
-
-namespace System.Diagnostics.CodeAnalysis
-{
-	/// <summary>
-	/// Indicates that the specified method requires the ability to generate new code at runtime,
-	/// for example through <see cref="System.Reflection"/>.
-	/// </summary>
-	/// <remarks>
-	/// This allows tools to understand which methods are unsafe to call when compiling ahead of time.
-	/// </remarks>
-	[AttributeUsage (AttributeTargets.Method | AttributeTargets.Constructor | AttributeTargets.Class, Inherited = false)]
-	public sealed class RequiresDynamicCodeAttribute : Attribute
```

Comment:
We can't really keep it because then there's two of them and the compiler doesn't know which one to pick.
I think we should basically bump the SDK to Preview3 in this change - ahead of Arcade. It should not cause much trouble - I just didn't get to to it yet today.
If should also resolve the RegEx problem as you mention below.